packages/client/lib/Chain.js

```diff
@@ -226,4 +226,12 @@ export default class Chain {
   async getAddressMempool (addresses) {
     return this.client.getMethod('getAddressMempool')(addresses)
   }
+
+  async getCode (address, block) {
```

Comment:
The namespaces should only include functions that can be reasonably implemented cross chain. 
You can still access the `getCode` function without it being in the namespace by calling `client.getMethod('getCode')` or through calling the provider directly. 
It's fine to have it in the provider if you are building another provider that wants to access it.

Reply:
There could be the case for Bitcoin, where `getCode` is used to get the bitcoin script used to redeem a p2sh. Might be a bit far fetched though.

Reply:
I've removed this for now.